timeline/views_ajax.py

```diff
@@ -36,13 +36,30 @@ def get_event_details(self, event):
             'status': [],
         }
         status_changes = event.get_status_changes(reverse=True)
-        for s in status_changes:
+        for idx, s in enumerate(status_changes):
+            desc = ''
+            if s.extra_data:
+                desc = (
+                    '<a class="sodar-tl-link-status-extra-data text-primary pull-right" data-url="{}">'
```

Comment:
We should form this HTML in the JQuery instead. We generally don't want to push preformatted HTML data to the client side in Ajax view.

The JSON formatting was an exception since it was really the best approach, but in the case of this simple link we should be able to easily create it in JQuery. Just returning the URL should be sufficient.

Reply:
Moved logic to js